lib/hash_ring.ex

```diff
@@ -67,11 +73,18 @@ defmodule ExHashRing.HashRing do
       when num > 0 and map_size(overrides) > 0 do
     {found, found_length} =
       case overrides do
-        %{^key => override} -> {[override], 1}
+        %{^key => overrides} -> Utils.take_max(overrides, num)
```

Comment:
Do we want Utils.take_max to take from the end of the list instead of the beginning of the list since we reverse on line 342 of ets.ex?

Reply:
yes, you're right :joy:.

will fix this up and add some tests.